packages/calypso-e2e/src/lib/components/navbar-component.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Internal dependencies
+ */
+import { BaseContainer } from '../base-container';
+
+/**
+ * Type dependencies
+ */
+import { Page } from 'playwright';
+
+/**
+ * Component representing the navbar/masterbar at top of WPCOM.
+ *
+ * @augments {BaseContainer}
+ */
+export class NavbarComponent extends BaseContainer {
+	// Selectors
+	navBarSelector = '.masterbar';
+	newPostButtonSelector = '.masterbar__item-new';
+	newPostContentSelector = '.masterbar__item-new';
```

Comment:
These two above have the same value. Is that intended?

Reply:
Oops, that's my mistake!